src/webui/api/torrentscontroller.cpp

```diff
@@ -31,9 +31,11 @@
 #include <functional>
 
 #include <QBitArray>
+#include <QHash>
 #include <QJsonArray>
 #include <QJsonObject>
 #include <QList>
+#include <QNetworkCookie>
```

Comment:
Is it required to include it? I didn't found reference to it.

Reply:
I had removed `/fetchMetadata`'s cookie support (which is no longer relevant) in the wrong commit, so this was needed when building each commit individually. Should be fixed now.